aws-py-eks/iam.py

```diff
@@ -1,72 +1,73 @@
 from pulumi_aws import config, iam
 import json
 
-## EKS Cluster Role
+def setup_iam():
```

Comment:
This is sort of introducing another new kind of "nesting" - beyond:
1. Components
2. Python modules

I suspect in practice we would want to encourage refactoring into components in these cases?